tracer/build/_build/Build.Steps.cs

```diff
@@ -1451,7 +1451,7 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     //.WithMemoryDumpAfter(timeoutInMinutes: 30)
                     .EnableNoRestore()
                     .EnableNoBuild()
-                    .SetFilter(string.IsNullOrWhiteSpace(Filter) ? "RunOnWindows=True&Category=AzureFunctions" : Filter)
+                    .SetFilter("RunOnWindows=True&Category=AzureFunctions")
```

Comment:
Downside to this is you can't test a single azure function sample now, which you _always_ want to do when you're running locally (because you can't test against multiple AzFunc Versions concurrently locally) 🤔

Reply:
Reverted in 9c61e57cfeaf14a61b9e24081d47931654d23fd6 . If we run into troubles with this again we can revisit 😄